From dcba56cafa8e3917fc1bfb9f295c3f23555663b9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 5 Jan 2012 14:48:57 +0100
Subject: [PATCH] Makefile: dash does not like {} expansion

removed {} expansion for mandir mkdir for users, which have dash as
/bin/sh
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index ccabd1e..e331c5f 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ install: doc
 	mkdir -p $(DESTDIR)$(bindir)
 	mkdir -p $(DESTDIR)$(sysconfdir)
 	mkdir -p $(DESTDIR)$(pkglibdir)/modules.d
-	mkdir -p $(DESTDIR)$(mandir)/man{5,7,8}
+	mkdir -p $(DESTDIR)$(mandir)/man5 $(DESTDIR)$(mandir)/man7 $(DESTDIR)$(mandir)/man8
 	install -m 0755 dracut $(DESTDIR)$(bindir)/dracut
 	install -m 0755 dracut-gencmdline $(DESTDIR)$(bindir)/dracut-gencmdline
 	install -m 0755 dracut-catimages $(DESTDIR)$(bindir)/dracut-catimages
